plugins/ptp/libptp/ptp.c

```diff
@@ -262,10 +262,10 @@ ptp_init_send_memory_handler(PTPDataHandler *handler,
 	unsigned char *data, unsigned long len
 ) {
 	PTPMemHandlerPrivate* priv;
+	handler->priv = priv;
```

Comment:
This doesn't make sense. priv is undefined at this point.

Reply:
Yeah, the two lines should be swapped.

Reply:
@korli @owenca I must have misunderstood Owen's comment. Now I see the whole point of the recommended fix.